### PR TITLE
python3Packages.labmath: init at 1.2.0

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
```ShellSession
$ nix-shell -I nixpkgs=. -p "python3.withPackages (p: [ p.labmath ])" \
    --command "python3 -c 'from labmath import *; print([totient(n) for n in range(1, 24)])'"
[1, 1, 2, 2, 4, 2, 6, 4, 6, 4, 10, 4, 12, 6, 8, 8, 16, 6, 18, 8, 12, 10, 22]
```

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 117882` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.labmath</li>
    <li>python39Packages.labmath</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117882` at 6c912144 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.labmath</li>
    <li>python39Packages.labmath</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/labmath/default.nix:20:0:
```
   |
20 |     description = "Module for basic math in the general vicinity of computational number theory";
   | ^
```
</li>
  </ul>
</details>

**Comment by risicle** [comment]
I saw surprised this kind of package appears to have no tests. Scary. Digging a bit it looks like a lot of the docstrings are written in a form possibly digestible by `doctests`, so as an improvement we could try running `doctests` over it.

But anyway, WFM linux x86_64.

**Review by risicle — APPROVED** [review]

**Comment by rmcgibbo** [comment]
Result of `nixpkgs-review pr 117882` at 6c912144 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.labmath</li>
    <li>python39Packages.labmath</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/labmath/default.nix:20:0:
```
   |
20 |     description = "Module for basic math in the general vicinity of computational number theory";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/labmath/default.nix:20:0:
```
   |
20 |     description = "Module for basic math in the general vicinity of computational number theory";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/labmath/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  '';
+
+  pythonImportsCheck = [ "labmath" ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/labmath";
+    description = "Module for basic math in the general vicinity of computational number theory";
+    license = licenses.mit;
+    maintainers = with maintainers; [ siraben ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/labmath/default.nix:23]:
```suggestion
```

set by buildPythonPackage to the platforms python supports.

**Pull request merged**